governance/polls/PPG - Open Market Committee Proposal - November 28, 2022.md

```diff
@@ -0,0 +1,80 @@
+* Reduce the WBTC-B [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 500 million DAI to **250 million DAI**.
+* Reduce the WBTC-B [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 30 million DAI to **10 million DAI**.
+* Increase the WBTC-B [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) from 8 hours to **24 hours**.
+* Reduce the WBTC-C [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1 billion DAI to **500 million DAI**.
+* Reduce the WBTC-C [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 100 million DAI to **20 million DAI**.
+* Increase the WBTC-C [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) from 8 hours to **24 hours**.
+* Reduce the LINK-A [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 7 million DAI to **2.5 million DAI**.
+* Reduce the MANA-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1 million DAI to **0 DAI**.
+* Reduce the YFI-A [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) from 7 million DAI to **1.5 million DAI**.
+* Reduce the Compound V2 D3M [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 160 million DAI to **10 million DAI**.
+* Reduce the GUNIV3DAIUSDC1-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1 billion DAI to **100 million DAI**.
+* Reduce the GUNIV3DAIUSDC2-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 1.25 billion DAI to **100 million DAI**.
+* Reduce the UNIV2DAIUSDC-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 300 million DAI to **100 million DAI**.
+* Increase the PSM-USDP-A [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) from 0% to **0.1%**.
+* Reduce the PSM-USDP-A [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 500 million DAI to **450 million DAI**.
+* Increase the PSM-GUSD-A [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) from 0% to **0.1%**.
+* Reduce the PSM-GUSD-A [Fee Out (`tout`)](https://manual.makerdao.com/module-index/module-psm#fee-out-tout) from 0.2% to **0.1%**.
```

Comment:
There are a *lot* of changes here. Perhaps try grouping the changes to make it more readable. See this poll for an example of what I mean - https://vote.makerdao.com/polling/QmfMRfE4#poll-detail